monitoring/monit.py

```diff
@@ -86,8 +89,35 @@ def run_command(command):
         module.run_command('%s %s %s' % (MONIT, command, name), check_rc=True)
         return status()
 
-    process_status = status()
-    present = process_status != ''
+    def wait_for_monit_to_stop_pending():
+        """Fails this run if there is no status or it's pending/initalizing for timeout"""
+        start_time = time.time()
+        timeout_time = start_time + timeout
```

Comment:
You can do `timeout_time = time.time() + timeout`

These are simple expressions, so fitting it into one line makes it a tad more readable.